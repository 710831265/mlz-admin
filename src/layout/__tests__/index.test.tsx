import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../tests';
import List from '..';

describe('ðŸ§ª List', () => {
  testMount(List);
  testSnapshot(List);
});
