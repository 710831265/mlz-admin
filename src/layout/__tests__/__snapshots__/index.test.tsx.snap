// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`🧪 List snapshot test 快照匹配 1`] = `"<section class=\\"ant-layout\\"></section>"`;
